mhs/common/mhs_common/workflow/tests/test_synchronous.py

```diff
@@ -212,7 +212,7 @@ def setUp(self) -> None:
 
         wdo.set_outbound_status.assert_called_with(work_description.MessageStatus.OUTBOUND_MESSAGE_RESPONSE_RECEIVED)
         self.assertEqual(error, 500)
-        self.assertEqual(text, 'Error(s) received from Spine: HTTP 451: Unknown')
+        self.assertEqual(text[:40], 'Error(s) received from Spine: HTTP 451: ')
```

Comment:
Because running this locally returned different error text but the status code is the same correct 451 one